region/brinstar/pink.json

```diff
@@ -1245,12 +1254,51 @@
                       "Grapple",
                       "h_canFly",
                       "canWalljump",
-                        {"and": [
-                          "SpeedBooster",
-                          "HiJump"
-                        ]}
+                      {"and": [
+                        "SpeedBooster",
+                        "HiJump"
+                      ]}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "Shinespark",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      {"canShineCharge": {
+                        "usedTiles": 16,
+                        "openEnd": 1,
+                        "shinesparkFrames": 24,
+                        "excessShinesparkFrames": 4
+                      }},
+                      {"canComeInCharged": {
+                        "fromNode": 3,
+                        "framesRemaining": 180,
+                        "shinesparkFrames": 38,
+                        "excessShinesparkFrames": 7
+                      }},
+                      {"canShineCharge": {
+                        "usedTiles": 24,
+                        "openEnd": 1,
+                        "shinesparkFrames": 38,
+                        "excessShinesparkFrames": 7
+                      }}
                     ]}
                   ]
+                },
+                {
+                  "name": "Big Pink Guide Enemy to Freeze",
```

Comment:
I think this notable name should be a bit more clear where it's going. as not to be confused with guiding a frozen enemy to ice clip to 14

Reply:
I'm wondering if this should maybe not be notable, since it's just to bypass a wall jump, and has `canTrickyUseFrozenEnemies`